client/asset/btc/electrum/jsonrpc_test.go

```diff
@@ -0,0 +1,41 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package electrum
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_floatString_UnmarshalJSON(t *testing.T) {
+	tests := []struct {
+		name    string
+		arg     []byte
+		wantErr bool
+	}{
+		{
+			name: "float64",
+			arg:  []byte{49, 50, 46, 50, 51}, // 12.23
+		},
+		{
+			name: "string float64",
+			arg:  []byte{34, 49, 50, 46, 50, 51, 34}, // "12.23"
```

Comment:
nit: The test would be easier to read if you just did json.Marshal inside the test and made these an interface or any or whatever and put the actual values in.

Reply:
Done.